pkg/asset/manifests/openshift.go

```diff
@@ -159,6 +160,15 @@ func (o *Openshift) Generate(dependencies asset.Parents) error {
 			cloud.CACertFile = "/etc/kubernetes/static-pod-resources/configmaps/cloud-config/ca-bundle.pem"
 		}
 
+		// Application credentials are easily rotated in the event of a leak and should be preferred. Encourage their use.
+		if cloud.AuthInfo != nil && (cloud.AuthType == clientconfig.AuthPassword || cloud.AuthType == clientconfig.AuthV2Password || cloud.AuthType == clientconfig.AuthV3Password) {
```

Comment:
This check is relatively early in install process, if the user catches it in time then they could easily change it. Should we consider `logrus.Warnf` so that the message is easier to find?

Reply:
Changed the log level to a warning. PTAL.